src/pages/en/guides/typescript.md

```diff
@@ -106,19 +106,42 @@ const { greeting = 'Hello', name } = Astro.props;
 <h2>{greeting}, {name}!</h2>
 ```
 
-### Common prop type patterns
+## Type Utilities
 
```

Comment:
Could we add a `<Since v="" />` component here to flag when these were added in case someone on an older version tries to use these?

Reply:
Yep, great idea!